db/data.py

```diff
@@ -59,6 +59,53 @@ def _has_key(dictionary, key):
     return True
 
 
+def show_failed_rows():
+    """
```

Comment:
A better description of what each of the values in the tuple returned are would be great.

For example: it isn't clear what offset means here.

Reply:
I have provided the detailed docstring about the values of the tuple in `show_failed_rows` command in `manage_highlevel.py` , should i provide it again here?

Reply:
yes, because it is hard to keep track of where the function is used to get info about what it returns.

Reply:
Right, okay, i'll do that